lib/api/redfish-1.0/event-service.js

```diff
@@ -237,7 +237,7 @@ var testEvent = controller(function(req, res) {
         });
     })
     .then(function() {
-        res.status(200);
+        res.status(202);
```

Comment:
Good catch!  The unit test was still checking for 200, and this was still returning 200.  Updated response code in controller flags and corresponding unit test.